### DISCUSSION TITLE
Looking for a solution

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#21880] [Devices]
i have a non zigbee device with dry contacts as an output and i want to transfer this signals to zigbee network to trifgger an action on a device or just receive this signals to mqtt for further automation and trigger a device. Use case: 

Video interkom panel (hikvision) with 4 dry  contacts -> wiring to a device "like zigbee lighting switch" -> received mqqt message when dry contat output was triggered from videopanel -> Trigger a zigbee door opener. 

Any ideas what device can i adopt to make it happen? i was thinking about small sonoff zigbee lighting switch and wire this dry contacts to "switch" signals, and leave Lout not connected but as far as i know they need to have 230 to make it work. 
any ideas welcome

**Comment by TheNutHouse84** [comment]
Zigbee Switch with dry contact output wired across switch terminals.[ Like this one ](https://www.aliexpress.us/item/3256806253578711.html) 

Does require basic soldering skills. Basically connect the dry contact output of your device to the mechanical pushbutton inside the zigbee switch. Cat 3/4/5 solid ethernet cable (strip out 2 wires) would work great for this. When the contact closes it will simulate a button press on the zigbee switch. The zigbee switch I listed does work with Z2M.
